x/pool-incentives/keeper/genesis.go

```diff
@@ -28,7 +28,7 @@ func (k Keeper) InitGenesis(ctx sdk.Context, genState *types.GenesisState) {
 	}
 	if genState.ConcentratedPoolToNoLockGauges != nil {
 		for _, record := range genState.ConcentratedPoolToNoLockGauges.PoolToGauge {
-			k.SetPoolGaugeIdNoLock(ctx, record.PoolId, record.GaugeId)
+			k.SetPoolGaugeIdNoLock(ctx, record.PoolId, record.GaugeId, record.Duration)
```

Comment:
Are we planning for any migration PR after this? 👀

Reply:
We shouldn't need to - please see #7375 (if you see any problems with the reasoning in that issue please flag them!)